sass/index.scss

```diff
@@ -269,6 +269,16 @@ section {
 
 // FAQ
 #faq {
+    #faq-title {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+
+        #faq-title-content {
+            margin-right: 15px;
+        }
+    }
```

Comment:
I'm a little conflicted on having nested ID selectors. It's better from a code readability perspective, but it does slightly increase runtime / download size for no reason; since `#faq-title`, `#faq-title-content`, etc. are unique across the page, generating the selector `#faq #faq-title #faq-title-content` is wasteful when just `#faq-title-content` would suffice.

The impact isn't bad now, but we should keep it in mind, especially since things like nested comma selectors can make this problem worse (e.g. `#a { #b { #c { .one, .two { .. } } } }` expands to `#a #b #c .one` and `#a #b #c .two` instead of `#c .one` and `#c .two`).

Reply:
I have mostly been considering code readability here. It might be contributing to the slow load time, but I am four prs in before seeing this and that's going to be quite a bit to refactor considering the time frame and how busy I currently am

Reply:
I really think runtime/download size for generating selectors is kinda losing the trees from the forest; especially on our tight deadline i'd argue code size on compile is a lower priority than optimizing page load (on the topic of optimizations)

Reply:
That's completely fair. I don't think this project is large enough for it to be a meaningful issue; I just tend to focus on download sizes a lot.